code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -125,6 +125,8 @@
 		/obj/item/weapon/card/debit/preferred/department/security,
 		/obj/item/weapon/pinpointer,
 		/obj/item/weapon/storage/box/large/securitygear,
+		/obj/item/clothing/suit/armor/hos/surveyor,
+		/obj/item/clothing/head/HoS/surveyor,
```

Comment:
It should go into the pick() on line 110 as a list of items to spawn since those are the alt. outfits. Or be the brave soul to add the outfits locker to HoS offices.

Reply:
Nah